pythonsd/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class Organizer(models.Model):
+    """Meetup organizers - displayed on the organizers page."""
+
+    name = models.CharField(max_length=255)
+    meetup_url = models.URLField(max_length=255, blank=True)
+    linkedin_url = models.URLField(max_length=255, blank=True)
+    active = models.BooleanField(
+        default=True,
+        help_text="Set to False to hide this organizer from the organizers page",
+    )
+
+    # For production, store the image in Cloud Storage (S3, Appwrite, etc.)
+    photo = models.ImageField(
+        upload_to="organizers/",
+        help_text="Recommended size of 400*400px or larger square",
+    )
+
+    def __str__(self):
```

Comment:
```suggestion
    class Meta:
        indexes = [
            models.Index(
                fields=['active', 'name'], 
                name='organizer_active_name_idx',
            ),
        ]

    def __str__(self):
```

This might be a bit overkill for how few organizers there are, but this is what I would do if the QuerySet was larger.

Reply:
I hope you don't mind, but I'll probably skip this. I think we can index just "active" if necessary but I'm expecting the table to be tiny. Normally, I'd probably just index "active" and rely on Postgres' ability to use multiple indexes rather than creating a composite index (which you'd do for MySQL unless my knowledge is outdated).

Reply:
I definitely don't mind, just a premature optimization suggestion.